front/components/cart/cart_controller.js

```diff
@@ -24,8 +15,4 @@ function($scope, ajaxUtil){
 
   $scope.total=$scope.getTotal();
```

Comment:
This is not a good idea because you are only getting the total once.  It is very possible that the cart list will change and you need to get the total again.  To handle this, you can get the total in the partial.